universal-application-tool-0.0.1/tailwind.config.js

```diff
@@ -1,43 +1,145 @@
+const fs = require('fs')
+const assert = require('assert')
+
+/* IMPORTANT
+ *
+ * If you modify this file, make sure to modify any comments in a
+ *    `app/views/style/Styles.java`
+ *    `app/views/style/ReferenceClasses.java`
+ *    `app/views/style/README.md
+ */
+
+// For stylesDict from Styles.java and ReferenceClasses.java
+const JAVA_STYLE_KEY_REGEX =
+  /(?<= +public +static +final +String +)([0-9A-Z_]+)/g
+const JAVA_STYLE_VALUE_REGEX =
+  /(?<= +public +static +final +String +[0-9A-Z_]+ += +")([a-z0-9-/.]+)/g
+
+const STYLE_USAGE_REGEX = /(?<=(Styles|ReferenceClasses)\.)([0-9A-Z_]+)/g
+
+let MEDIA_QUERY_CALL_STR_START = '/(?<=StyleUtils.('
+
+const MEDIA_QUERY_BEGIN = '(?<=StyleUtils.'
+const MEDIA_QUERY_CALL_END = '\\()([a-zA-Z0-9_.,\\s]+\\))'
+
+// Files to parse for style dictonary using regex
+const styleFolder = './app/views/style/'
+const styleFiles = ['Styles.java', 'ReferenceClasses.java']
+
+// Prefixes for media queries
+const PREFIXES = [
+  ['even', 'even'],
+  ['focus', 'focus'],
+  ['focusWithin', 'focus-within'],
+  ['hover', 'hover'],
+  ['disabled', 'disabled'],
+  ['responsiveSmall', 'sm'],
+  ['responsiveMedium', 'md'],
+  ['responsiveLarge', 'lg'],
+  ['responsiveXLarge', 'xl'],
+  ['responsive2XLarge', '2xl'],
+]
+
+// Used to read Styles.java and ReferenceClasses.java to get dictionary mapping
+// for all possible base styles (no prefixes)
+function addStyleDictMatches(matches, file_contents) {
+  let lineNmbr = 0
+  for (const line of file_contents) {
+    lineNmbr++
+
+    let match_key = line.match(JAVA_STYLE_KEY_REGEX)
```

Comment:
These need to be camel cased.  We use the google style guides.

https://google.github.io/styleguide/jsguide.html#naming-rules-common-to-all-identifiers